src/main/scala/no/ndla/articleapi/service/converters/HTMLCleaner.scala

```diff
@@ -271,6 +272,15 @@ trait HTMLCleaner {
       element
     }
 
+    private def unwrapDivsInAsideTags(element: Element) = {
+      val asides = element.select("body>section>aside").asScala
+      for (aside <- asides) {
+        if (aside.children.size == 1 && aside.children.first.tagName == "div") {
```

Comment:
I eksemplet i trello boardet er det kun én div inne i en aside, men jeg kan se for meg at det finnes tilfeller hvor det ligger flere nøstede divs også. Tror det er lurt å ta høyde for det.